client/src/components/History/Content/ContentOptions.vue

```diff
@@ -58,7 +58,7 @@ export default {
         isDataset: { type: Boolean, required: true },
         isDeleted: { type: Boolean, required: true },
         isHistoryItem: { type: Boolean, required: true },
-        isPurged: { type: Boolean, required: true },
+        isPurged: { type: Boolean, default: false },
```

Comment:
Trying to figure out more context for this change -- is there a good reason to loosen the requirement here and add a default instead of an explicitly required prop?

Reply:
Yes, I was overly restrictive here when I separated this component by requiring all props, but it turns out that collections do not provide the `purged` flag.